src/driver/communication_interface.cc

```diff
@@ -408,6 +426,8 @@ void CommunicationInterface::HandlePlan(
   }
 
   new_plan_buffer_.utime = robot_spline->utime;
+  new_plan_buffer_.exec_opt = robot_spline->exec_opt;
```

Comment:
have we updated `robot_spline_t` definition?

Reply:
Don't need to. This was added to `robot_spline_t` a while ago for bowl passing with the AA